.github/workflows/invalidate-prs.yaml

```diff
@@ -0,0 +1,26 @@
+# This workflow invalidates the builds of pull requests that more than 3 hours old.
+#
+
+name: "Invalidate PRs after 3 hours"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
```

Comment:
Can you add to this comment that the pattern defines that it runs ever 15 minutes.

Reply:
I think @wgoehrig mentioned about internal wiki page.